json_templates/azure/c0-azure-base/ert-template.json

```diff
@@ -65,46 +65,325 @@
-      "ccdb": {"instance_type": {"id": "automatic"},"instances": 0,"elb_names": [],"persistent_disk":{"size_mb":"automatic"}},
-      "uaadb": {"instance_type": {"id": "automatic"},"instances": 0,"elb_names": [],"persistent_disk":{"size_mb":"automatic"}},
-      "uaa": {"instance_type": {"id": "automatic"},"instances": 2,"elb_names": []},
-      "cloud_controller": {"instance_type": {"id": "automatic"},"instances": 2,"elb_names": [],"persistent_disk":{"size_mb":"automatic"}},
-      "ha_proxy": {"instance_type": {"id": "automatic"},"instances": 0,"elb_names": []},
-      "router": {"instance_type": {"id": "automatic"},"instances": 2,"elb_names": ["{{terraform_prefix}}-web-lb"]},
-      "mysql_monitor": {"instance_type": {"id": "automatic"},"instances": 1,"elb_names": []},
-      "clock_global": {"instance_type": {"id": "automatic"},"instances": 1,"elb_names": []},
-      "cloud_controller_worker": {"instance_type": {"id": "automatic"},"instances": 3,"elb_names": []},
-      "collector": {"instance_type": {"id": "automatic"},"instances": 0,"elb_names": []},
-      "diego_database": {"instance_type": {"id": "automatic"},"instances": 3,"elb_names": [],"persistent_disk":{"size_mb":"automatic"}},
-      "diego_brain": {"instance_type": {"id": "automatic"},"instances": 2,"elb_names": ["{{terraform_prefix}}-ssh-proxy-lb"],"persistent_disk":{"size_mb":"automatic"}},
-      "diego_cell": {"instance_type": {"id": "automatic"},"instances": 12,"elb_names": []},
-      "doppler": {"instance_type": {"id": "automatic"},"instances": 2,"elb_names": []},
-      "loggregator_trafficcontroller": {"instance_type": {"id": "automatic"},"instances": 3,"elb_names": []},
-      "tcp_router": {"instance_type": {"id": "automatic"},"instances": 2,"elb_names": ["{{terraform_prefix}}-tcp-lb"],"persistent_disk":{"size_mb":"automatic"}},
-      "backup-prepare": {"instance_type": {"id": "automatic"},"instances": 0,"elb_names": [],"persistent_disk":{"size_mb":"automatic"}},
-      "push-apps-manager": {"instance_type": {"id": "automatic"},"instances": 1,"elb_names": []},
-      "smoke-tests": {"instance_type": {"id": "automatic"},"instances": 1,"elb_names": []},
-      "notifications-ui": {"instance_type": {"id": "automatic"},"instances": 1,"elb_names": []},
-      "autoscaling-register-broker": {"instance_type": {"id": "automatic"},"instances": 1,"elb_names": []},
-      "autoscaling-destroy-broker": {"instance_type": {"id": "automatic"},"instances": 1,"elb_names": []},
-      "push-pivotal-account": {"instance_type": {"id": "automatic"},"instances": 1,"elb_names": []},
-      "mysql-rejoin-unsafe": {"instance_type": {"id": "automatic"},"instances": 1,"elb_names": []},
-      "autoscaling": {"instance_type": {"id": "automatic"},"instances": 1,"elb_names": []},
-      "bootstrap": {"instance_type": {"id": "automatic"},"instances": 1,"elb_names": []},
-      "notifications": {"instance_type": {"id": "automatic"},"instances": 1,"elb_names": []}
+    "consul_server": {
```

Comment:
@krishicks i have verified it using BeyondCompare text compare. It seems to be just formatting which has changed. there is no change in the text though.